File: Matlab/bench_2_1.m
function bench_2_1() 

trials = 100;
elements = 1000000;

times = zeros(1,trials);

for i = 1:trials

   tic();
   randi(999, 1, elements);
   t = toc();
   
   times(i) = t;
   
end

fprintf('Matlab - Benchmark 2.1: Average time = %f milliseconds\n', mean(times)*1000);

File: Python/bench_2_1.py
import numpy as np
from time import time

def bench_2_1():

    trials = 100
    elements = 1000000

    times = []

    for i in range(trials):

        start = time()
        M = np.random.randint(1,999, size=elements)
        t = time()-start

        times.append(t)

    print 'Python - Benchmark 2.1: Average time = {} milliseconds'.format(np.mean(times)*1000)